code/pom.xml

```diff
@@ -33,6 +33,31 @@
         <artifactId>appengine-api-1.0-sdk</artifactId>
         <version>1.9.59</version>
     </dependency>
+    <dependency>
+      <groupId>com.googlecode.objectify</groupId>
+      <artifactId>objectify</artifactId>
+      <version>6.0.6</version>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+    </dependency>
+    <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Let's try to use GSON (the google provided JSON parsing library) rather than Jackson (which I have no idea who makes it)

http://tutorials.jenkov.com/java-json/gson-installation.html